creativecloud/scripts/utils.js

```diff
@@ -61,7 +61,7 @@ function getDecorateAreaFn() {
   function replaceDotMedia(area = document) {
     const currUrl = new URL(window.location);
     const pathSeg = currUrl.pathname.split('/').length;
-    if (pathSeg >= 3) return;
+    if (pathSeg >= 4) return;
```

Comment:
This logic seems for skipping media updates if the URL isn't root path.
In fact, it will still work even if it isn't root path since the media will work regardless the sub-path as long as it listen to the Akamai correctly. But I just increased 1 for checking and left the original skipping logic instead removing it since it won't hurt to skip anyway.

Reply:
Agreed with the statement. We should only run it for root page for en and locale/root for corresponding locale page.
@salonijain3 @amitbikram Could you share your inputs on the same?